strongbox-storage/strongbox-storage-api/pom.xml

```diff
@@ -288,6 +288,11 @@
             <groupId>org.javatuples</groupId>
             <artifactId>javatuples</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.semver</groupId>
+            <artifactId>api</artifactId>
```

Comment:
Under what license is this?

Reply:
it's from Maven Central, Does not that mean the right license?

Reply:
You can check this on Github, or in the `pom.xml` (although, this wasn't described there).

[Here is the license](https://github.com/jeluard/semantic-versioning/blob/master/LICENSE). It's Apache 2.0.

Please, use Maven Central's "advanced search" to find out details regarding licenses when you're adding new dependencies. We should try to stick to Apache 2.0, or MIT licensed libraries.

Thanks! :)